internal/k8s/crdcontroller.go

```diff
@@ -794,6 +794,11 @@ func validateHostRuleObj(key string, hostrule *akov1alpha1.HostRule) error {
 		return err
 	}
 
+	// During ako reboot the hostrule is validated by the old AKO, hence returning.
```

Comment:
could you change the comment to better reflect updated the workflow - something like "no need to update status to accepted if it was accepted before".
Also, could you please add this logic for AviInfraSetting CRD too.